backend/notification_v2/clubbed_renderer.py

```
@@ -0,0 +1,201 @@
+"""Canonical envelope + renderer for every dispatch — IMMEDIATE and BATCHED.
+
+The same envelope shape feeds every channel × mode cell so receivers never
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)